Sources/NetworkProtection/Routing/VPNRoutingRange.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  VPNRoutingRange.swift
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public enum VPNRoutingRange {
+
+    public static let alwaysExcludedIPv4Range: [NetworkProtection.IPAddressRange] = [
+        "127.0.0.1/8",    /* 255.0.0.0   Loopback */
+        "169.254.0.1/16", /* 255.255.0.0 Link-local */
+        "224.0.0.1/4",    /* 240.0.0.0   Multicast */
+        "240.0.0.1/4",    /* 240.0.0.0   Class E */
+    ]
+
+    public static let alwaysExcludedIPv6Range: [NetworkProtection.IPAddressRange] = [
+        "fe80::/10",  /* link local */
+        "ff00::/8",   /* multicast */
+        "fc00::/7",   /* local unicast */
+        "::1/128",    /* loopback */
+    ]
+
+    public static let localNetworkRange: [NetworkProtection.IPAddressRange] = [
+        // "10.0.0.0/8",     /* 255.0.0.0   */
```

Comment:
Is this commented out because it clashes with our VPN tunnel IP range? Should we add ranges surrounding the tunnel range?

Reply:
I'm matching Android here, where they exclude the 10.x.x.x range because it's used for DNS and generally reserved for out VPN.

I do think we should included everything but the DNS but since Android is doing this, I'm assuming it's good enough for us, for now.